docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+---
+version: '3.7'
+services:
+
+  readeef:
+    image: readeef:alpine3.10-go1.13
+    container_name: readeef
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - ${READEEF_PORT}:8080
+    depends_on:
+    - db
+    - elasticsearch
+    links:
+    - db
+    - elasticsearch
+    volumes:
+    - .docker/readeef/config/readeef.toml:/opt/readeef/config/readeef.toml
+    command: ["/opt/readeef/bin/readeef", "-config=/opt/readeef/config/readeef.toml", "server"]
+    #restart: unless-stopped
+
+  rfng:
+    image: rf-ng:alpine3.10-go1.13
```

Comment:
does this container exist?

Reply:
yes, I created a dockerfile to build it. It is in rf-ng dir. As I still struggle to understand how it works I made one anyway. ^^